rewrite-core/src/main/java/org/openrewrite/config/YamlResourceLoader.java

```diff
@@ -167,17 +169,19 @@ public YamlResourceLoader(InputStream yamlInput, URI source, Properties properti
 
     private Collection<Map<String, Object>> loadResources(ResourceType resourceType) {
         Collection<Map<String, Object>> resources = new ArrayList<>();
-
         Yaml yaml = new Yaml(new SafeConstructor(new LoaderOptions()));
-        for (Object resource : yaml.loadAll(yamlSource)) {
-            if (resource instanceof Map) {
-                @SuppressWarnings("unchecked") Map<String, Object> resourceMap = (Map<String, Object>) resource;
-                if (resourceType.equals(ResourceType.fromSpec((String) resourceMap.get("type")))) {
-                    resources.add(resourceMap);
+        try {
+            for (Object resource : yaml.loadAll(yamlSource)) {
+                if (resource instanceof Map) {
+                    @SuppressWarnings("unchecked") Map<String, Object> resourceMap = (Map<String, Object>) resource;
+                    if (resourceType.equals(ResourceType.fromSpec((String) resourceMap.get("type")))) {
+                        resources.add(resourceMap);
+                    }
                 }
             }
+        } catch (Exception e) {
+            logger.error("Loading yaml {} type failed, yaml source: {}", resourceType, yamlSource);
```

Comment:
should we print the error message as well to understand what is the parsing error?